README.md

```diff
@@ -1 +1,8 @@
-# git-playground
\ No newline at end of file
+# git-playground
+
+# Hi there
+
+#Second Change
+
+
+#third
```

Comment:
This line isn't up to standards, we always put a space between '#' and the text.